pkgs/applications/blockchains/pivx.nix

```diff
@@ -56,9 +55,5 @@ stdenv.mkDerivation rec {
     homepage = "https://www.dash.org";
     maintainers = with maintainers; [ wucke13 ];
     platforms = platforms.unix;
-    # TODO
-    # upstream doesn't support newer openssl versions
-    # https://github.com/PIVX-Project/PIVX/issues/748
-    # openssl_1_0_2 should be replaced with openssl ASAP
```

Comment:
Nice!

Reply:
Finally, yes. This was a really bad situation...